acceptance/bundle/resources/apps/output.txt

```diff
@@ -9,16 +12,30 @@ Deployment complete!
 {
   "body": {
     "description": "my_app_description",
-    "name": "myapp"
+    "name": "myappname"
   },
   "method": "POST",
   "path": "/api/2.0/apps"
 }
-apps myapp name='myapp' description='my_app_description'
+
+>>> [CLI] bundle summary
+Name: test-bundle
+Target: default
+Workspace:
+  User: [USERNAME]
+  Path: /Workspace/Users/[USERNAME]/.bundle/test-bundle/default
+Resources:
+  Apps:
+    mykey:
+      Name: myappname
+      URL:  (not deployed)
```

Comment:
Is it expected that URL is "(not deployed)" here? Is it because the app is not started? However, we already have the URL, we could show it here. @andrewnester